packages/react/src/components/radio/radio.stories.tsx

```diff
@@ -89,14 +90,14 @@ export const WithValidation = ({
   withIcon,
   ...restRadioProps
 }: RadioWithValidationProps) => (
-  <>
+  <Fieldset>
```

Comment:
This is probably better as a Field rather than a Fieldset, don't you think?

Unless this had a legend, or several Fields in it, so that you could do, for instance:

`<Fieldset disabled>`

Or is the intent to finish this story after `Legend` is done?

```ts
<Fieldset>
  <Legend />
  <Field>
     <Radio />
     <Validation />
```

Reply:
For us `Field` is not an option for Checkbox and Radio components - styling is not right, that is `width: min-content` breaks validation message.

Probably I will add a legend here, not sure - will see how it fits best at later PR.

Reply:
Hmm that might be tricky if `Field` is the one holding validation context later. We'll see.